java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_2/BinarySearchTree.java

```diff
@@ -0,0 +1,44 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class BinarySearchTree {
+    private Node rootNode;
+    public BinarySearchTree () {
+        rootNode = null;
+    }
+
+    private static class Node {
+        private final int value;
+        private Node leftChild;
+        private Node rightChild;
+        public Node (int value) {
+            this.value = value;
+            leftChild = null;
```

Comment:
same: leftNode and rightNode initialization statements can be removed

Reply:
Done